go/enclave/rollupchain/rollup_chain.go

```diff
@@ -725,12 +712,13 @@ func (rc *RollupChain) GetBalance(encryptedParams common.EncryptedParamsGetBalan
 	if err != nil {
 		return nil, fmt.Errorf("failed to unmarshal RPC request params from JSON. Cause: %w", err)
 	}
-	if len(paramList) < 2 {
-		return nil, fmt.Errorf("required exactly two params, but received zero")
+	if len(paramList) != 2 {
+		return nil, fmt.Errorf("required exactly two params, but received %d", len(paramList))
```

Comment:
https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getbalance

Needs to be precisely 2 args